framework/include/utils/InputParameters.h

```diff
@@ -659,27 +659,38 @@ class InputParameters : public Parameters
    */
   std::set<std::string> reservedValues(const std::string & name) const;
 
-private:
-  // Private constructor so that InputParameters can only be created in certain places.
-  InputParameters();
+  /// Get/set a string representing the location (i.e. filename,linenum) in the input text for the
+  /// block containing parameters for this object.
```

Comment:
This isn't the correct format for comments in MOOSE source files. We use the block format for methods and three-slashes for member variables. Also, so that Doxygen knows that two methods share the same comment you'll want to use the special "///@{ ///@}" syntax around these paired methods. Let's get in the habit of getting this correct. It makes the maintenance easier if we all follow the same conventions. If you make only comment changes, we won't need to wait for the testing cycle to complete before this can be merged.